extra-utils/hdsentinel/autobuild/build

```diff
@@ -0,0 +1,21 @@
+abinfo Detect architecture
+hdsentinelVer="017"
+hdsentinelBaseName="hdsentinel-$hdsentinelVer"
+if [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName-x64"
+elif [[ "${CROSS:-$ARCH}" = "i386" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName"
+elif [[ "${CROSS:-$ARCH}" = "armel" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName-arm"
+else
+    exit 1
+fi
+
+abinfo Downloading pre-built binaries
```

Comment:
Bin-pack is usually not preferred... bin-packs in our repository are mostly because we have no way to build them from source. (Although you can still do it as the license permits)

Reply:
No source available for `hdsentinel`. The only reason I'm packaging it is I'm used to it on Windows.